scriptmodules/supplementary/sixaxis.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="sixaxis"
+rp_module_desc="Helper service to install & configure the latest drivers for official and third-party DualShock controllers (ps3controller replacement)\n\nNote 1: this driver uses different button maps to the old driver, so you may need to remap your controller.\nNote 2: If you have a third-party controller (Shanwan/Gasia), you must also install 'custombluez'.\n\nTo pair controllers, enter the RetroPie Bluetooth menu, choose 'Register and Connect...', then connect controller via USB when the 'Scanning' prompt appears."
+rp_module_licence="GPL2 https://raw.githubusercontent.com/psyke83/sixaxis/master/COPYING"
+rp_module_section="driver"
+
+function depends_sixaxis() {
+    local depends=(checkinstall libevdev-tools)
+
+    getDepends "${depends[@]}"
+
+    rp_callModule ps3controller remove
+    rp_callModule customhidsony
+    if isPlatform "rpi"; then
```

Comment:
Are we going to continue supporting jessie? This package or an equivalent that has the needed functionality isn't available in the older distro. If we're dropping support, I could simply remove this whole section and make things a bit cleaner.

Reply:
I would like to continue it at least for a bit - but if we drop support just for one package I don't mind perhaps. Will have a think.